docs/methodology.md

```
@@ -0,0 +1,285 @@
+    B -->|Cold run| D[1.&nbspStart polling system metric<br> 2.&nbspSet start timestamp]
+    C --> D
+    D --> E[Execute Query]
+    E --> F[1.&nbspSet end timestamp<br> 2.&nbspStop polling system metric]
+    F --> Terminate
+```
+
+#### Runtime scenarios
+
+For search we evaluate two runtime scenarios for query benchmarking with each tool:
+
+* **Hot run**: In this scenario, we run the query multiple times to warm up any caches, then execute the query again to measure end-to-end latency.  
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)